### ISSUE TITLE
Stored authentication data for FreshRSS disappears every few weeks

### ISSUE THREAD

**Issue by psandtner** [Ranchero-Software/NetNewsWire#4408]
Use NetNewsWire in conjunction with FreshRSS. 

Every few weeks the stored data “Username”, “password” and the “API-Url” seem to “disappear” - with the effect that an “Alert” panel appears with the message that the authentication has failed. 

As soon as I enter all three data again (in unchanged form), it works again.

**Comment by psandtner** [comment]
The problem still exists for me. I have created a bug report with a screenshot via the “Beta feedback”.

**Comment by psandtner** [comment]
I have the impression that I am apparently the only one with this problem. 🫣

**Comment by mattegger** [comment]
> I have the impression that I am apparently the only one with this problem. 🫣

I seem to be having this issue as well, but only starting recently. Over the last 2 or 3 weeks, I've had to reenter my credentials and API URL twice. I haven't had time to do much digging

**Comment by constcast** [comment]
I have the same problem with exactly the same symptoms as @psandtner.  The problems started recently, probably after upgrading to ios26. Before that I had no problems with disappearing authentication data.

Let me know if I can somehow provide any data that helps with debugging the issue.

**Comment by psandtner** [comment]
Update: Interestingly, I haven't had this problem for months (I'm using a different FreshRSS instance; version 1.27.1).

iOS: I just checked: Only the ‘API URL’ input field appears to be empty (contains the greyed-out placeholder value) – but everything works. 

macOS: No problems here either. However, the ‘API URL’ I use is visible here.

**Comment by constcast** [comment]
I only have problems on iOS. When everything works as expected, the API-URL field is empty but the password is set.  

When the problem occurs, then both the URL and the password field are empty

**Comment by koruptor** [comment]
I just want to chime in that I am also experiencing this bug on iOS (I'm on version 6.2 Build 6202).  It's fine on MacOS for me.
The authentication fails every week or so.  As @constcast mentioned, the credentials fields reset and are empty.

Here's the error that I see on my iPhone when this occurs:

> Error
> An error occurred while processing the
> "FreshRSS" account: The operation couldn't be completed.
> (Secrets.CredentialsError error 1.)

**Comment by ch4r13** [comment]
Yeah I have also this issue![image](https://github.com/user-attachments/assets/c8eedf00-77ab-49f5-a601-a8b636a24571)

**Comment by abustany** [comment]
This is happening using miniflux (through the google reader API) for me. miniflux does never expire tokens (the token is just a HMAC of the username and password, see [getAuthToken](https://github.com/miniflux/v2/blob/5db66fc2799b38dbe7f26c3a3c0c865b624ee39b/internal/googlereader/middleware.go#L190)). So it seems that the token gets lost on iOS somehow... Skimming through the code, I couldn't find an obvious scenario (eg the server being down and returning a 5xx error code) where NNW would clear the credentials and not renew them 🤔

**Comment by brentsimmons** [comment]
I think we’ve fixed this (fix coming in 7.0.1), so I’ll close this. We can reopen if it turns out not to be fixed.
